### PR TITLE
Add Spotify parser

### PR DESCRIPTION
**Pull request by chrisforrette**
Hi there! I'm not sure how open you are to PRs, but this package is super handy for a project I'm currently working on—thank you for building!—and I could use some new parsers and updates and such so I decided I'd open a first PR and see what you think.

This PR adds a Spotify embed parser using the code and options outlined here: https://developer.spotify.com/technologies/widgets/spotify-play-button/

I followed the style you applied to the Vimeo parser, the only difference being that I excluded the `int()`s so that you can enter percentage widths/heights.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -25,6 +25,24 @@ Usage
...
+**Spotify**
+
+::
+
+    Hey there, check out this rad new song:
+    [spotify uri=spotify:track:41eiwHEX8iegmqmS2cf7oX]
+
+Available parameters:
+
+* ``uri`` (required): The Spotify URI for a track, album, playlist, etc., that you'd like to embed. This can be found in the Spotify app with "Copy Spotify URI" buttons.
```
Comment (martey) [inline README.rst]: Can we wrap these lines? I'd like to keep the README under 80 characters.

**Review by martey — CHANGES_REQUESTED** [review]
Thanks for the pull request. I haven't used this package in a while, so it is missing contributor guidelines and automated tests.

The suggestions I have are relatively minor: make sure the README lines are wrapped, that `spotify.py` passes flake8, and to squash everything down into one comment.

**Comment by chrisforrette** [comment]
Hi @martey! This should be all set—I just did some line-wrapping and flake8 checks (though I'll mention that flake8 doesn't pass for a few other files that I didn't touch).

You mentioned "squash everything down into one comment"—did you mean into one commit? If so, I know there's the handy drop-down on GitHub now to "Squash and merge".

Let me know if you have any other feedback on this one—thank you!

**Comment by chrisforrette** [comment]
@martey By the way, I had noticed the lack of tests and I'm happy to help. I was thinking of using py.test and coverage unless you have any other preference?

**Pull request closed without merging**